src/components/Resources/Schema/ViewSchema.tsx

```diff
@@ -265,137 +265,137 @@ const ViewSchemas = () => {
+                            <Label htmlFor="credential-definition" value="Name" />
+                            <span className='text-red-600'>*</span>
+                          </div>
+                          <Field
+                            id="tagName"
+                            name="tagName"
+                            placeholder="Enter Credential definition"
+                            className="w-full bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"
+                            disabled={createloader}
+                          />
+                          {
+                            (formikHandlers?.errors?.tagName && formikHandlers?.touched?.tagName) &&
+                            <span className="text-red-500 text-xs">{formikHandlers?.errors?.tagName}</span>
+                          }
                         </div>
-                        <Field
-                          id="tagName"
-                          name="tagName"
-                          placeholder="Enter Credential definition"
-                          className="w-full bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"
-                          disabled={createloader}
-                        />
-                        {
-                          (formikHandlers?.errors?.tagName && formikHandlers?.touched?.tagName) &&
-                          <span className="text-red-500 text-xs">{formikHandlers?.errors?.tagName}</span>
-                        }
                       </div>
-                    </div>
 
-                    <div className='flex items-center'>
-                      <div className="custom-control custom-checkbox d-flex align-items-center pt-4 p-2">
-                        {/* <Field type="checkbox" id="Revocable" name="revocable" className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-800 focus:ring-2 dark:bg-gray-700 dark:border-gray-600" /> */}
-                        {/* <Label className="ml-2 text-sm font-medium text-gray-900 dark:text-gray-300" >
+                      <div className='flex items-center'>
+                        <div className="custom-control custom-checkbox d-flex align-items-center pt-4 p-2">
+                          {/* <Field type="checkbox" id="Revocable" name="revocable" className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-800 focus:ring-2 dark:bg-gray-700 dark:border-gray-600" /> */}
```

Comment:
please remove commented code

Reply:
The code needs to remain commented for the purpose of revocable, which is why it cannot be removed.